CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Master (unreleased)
 
 * Optinally ignore method names in the `describe` argument when running the `FilePath` cop. ([@bquorning][])
+* Change how `FilePath` converts alphanumeric class names into paths. ([@bquorning][])
```

Comment:
Could you change this to something like "Fix regression in how `FilePath` ..."? I believe you are changing the behavior back to how it was before I fudged it, right?